homeassistant/components/miele/sensor.py

```diff
@@ -553,22 +551,6 @@ def options(self) -> list[str]:
         )
 
 
-class MieleTypeSensor(MieleSensor):
-    """Representation of the program type sensor."""
-
-    @property
-    def native_value(self) -> StateType:
-        """Return the state of the sensor."""
-        ret_val = STATE_PROGRAM_TYPE.get(int(self.device.state_program_type))
-        if ret_val is None:
-            _LOGGER.debug(
-                "Unknown program type: %s on device type: %s",
-                self.device.state_program_type,
-                self.device.device_type,
-            )
-        return ret_val
-
-
```

Comment:
If this is to be removed, please address this in a separate PR, as it seems a breaking change. Also it is not mentioned in the PR description.

Reply:
See above